### DISCUSSION TITLE
🤔 [QUESTION] nextjs引入报错，只支持import格式

### DISCUSSION THREAD

**Discussion by xky0007** [antvis/G2Plot#3567] [Q&A]
### 🐛 Question description [Please make everyone to understand it]
nextjs引入示例报错
```
Server Error
Error: require() of ES Module /workspace/node_modules/d3-interpolate/src/index.js from /workspace/node_modules/@antv/g-base/lib/animate/timeline.js not supported.
Instead change the require of index.js in /workspace/node_modules/@antv/g-base/lib/animate/timeline.js to a dynamic import() which is available in all CommonJS modules.
```
### 💻 Link to minimal reproduction

[Please provide a link by forking these links [g2plot](https://codesandbox.io/s/g2plot-reproduction-template-gyej1) or GitHub repo. What is a minimal reproduction, and why is it required?](https://github.com/xky0007/determined-hodgkin-52gblv)

### 🚑 Any additional [like screenshots]

* **G2Plot Version**: `2.4.29`
* **Platform**: `CodeSandBox`

**Comment by hustcc** [comment]
这个需要麻烦你也到 d3-interpolate 去提交一下 issue，让作者知道这个问题的严重性，如果底层修改打包方式，上层都自动解决了。😭

**Reply by suwu150** [reply]
现在有临时解决方案吗 @hustcc

**Reply by DophinL** [reply]
> 这个需要麻烦你也到 d3-interpolate 去提交一下 issue，让作者知道这个问题的严重性，如果底层修改打包方式，上层都自动解决了。😭

没找到g-base是哪个github repo，npm上显示的是antvis/util，但看起来不是。

以及能否不依赖d3-interpolate？

现在next.js用户越来越多了，能否解决这个问题？

**Comment by DophinL** [comment]
同问，有临时解决方案吗？

**Comment by lamorttu** [comment]
+1，有临时解决方案吗？

**Reply by lamorttu** [reply]
尝试修改overrides，好像也无法解决
